src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -97,7 +97,7 @@ export class ArtistInfo extends Component<ArtistInfoProps, ArtistInfoState> {
     const imageUrl = get(this.props, p => image.cropped.url)
     const showArtistInsightsButton =
       (artist.exhibition_highlights &&
-        artist.exhibition_highlights.length > 0) ||
+        artist.exhibition_highlights.length > 1) ||
```

Comment:
Would be cool to keep this synched with the Selected Exhibitions component: https://github.com/ds300/reaction/blob/0eb9877c357fde2edb85091dfb3cc1e670d5bae4/src/Components/v2/SelectedExhibitions.tsx#L10

and then make it

```suggestion
        artist.exhibition_highlights.length >= MIN_EXHIBITIONS)
```

Reply:
How should I do that?

Reply:
export `MIN_EXHIBITIONS` from the file I linked to above, `SelectedExhibitions.tsx`, then import it in this file and try out the suggested change above